.github/workflows/ci-zk-toolbox-reusable.yml

```diff
@@ -115,6 +115,10 @@ jobs:
           --prover-db-url=postgres://postgres:notsecurepassword@postgres:5432 \
           --prover-db-name=zksync_prover_localhost_rollup
 
+      - name: Check Database
+        run: |
+          ci_run zk_supervisor database check-sqlx-data
+
```

Comment:
why you moved it?

Reply:
So that we don't have to initialize the ecosystem every time we lint code

Reply:
but, we need only run setup database. not the whole ecosystem

Reply:
`zk_supervisor database check-sqlx-data` needs a `secrets.yaml` which is provided by `ecosystem init`